lib/style.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Assigns style attributes in a style-src compliant way.
+ *
+ * @param {Element} element
+ * @param {...Object} styleSources
+ *
+ * @return {Element} the element
+ */
+export function assign(element, ...styleSources) {
+  const target = element.style;
+
+  styleSources.forEach(function(style) {
+    if (!style) {
+      return;
+    }
+
+    for (const atrribute in style) {
```

Comment:
Do we need to worry about prototype injection here? If that is the case, let us use our existing _forEach_ helper.

Reply:
It already went through a couple of security checks.

Reply:
True, this might be a problem. Not that big of an issue with style definitions I think, but it might pop up on an audit either way. Better to filter it out early :+1:

Reply:
Uses min-dash `forEach` with [c1788a5](https://github.com/bpmn-io/min-dom/pull/13/commits/c1788a5ca72010f48840abec09a275de9785cc80)